packages/allure-mocha/runtime.d.ts

```diff
@@ -1 +1,3 @@
-export { allure } from "./dist/MochaAllureReporter";
+export { MochaAllure } from "./dist/MochaAllure";
+export { MochaAllureReporter } from "./dist/MochaAllureReporter";
```

Comment:
Should these values be exported from `index.ts` instead of runtime?

Reply:
As in `index.ts` we have the following:
```
export = MochaAllureReporter;
```
TS doesn't allow us putting any other exports there.

Reply:
Understood, let's keep it this way